tasks/nonograms/README.md

```diff
@@ -0,0 +1,115 @@
+- create new branch `nonograms` from main. Create folder `nonograms` in the created branch where you will do source code
+- the **main** (or **master**) branch should be empty (contain only files like README.md, .gitignore or .github folder)
+- commits should represent the work history. Commit messages should follow the [guideline](https://www.conventionalcommits.org/en/v1.0.0/)
+- once the work is finished, create a pull request from a development branch to main (or master). This PR doesn't need to be merged
+- the pull request name should contain **the task name**
+- the pull request description should contain the following information:
+  - link to the task
+  - screenshot of your application (one would be enough)
+  - link to your application
+  - date of completion/deadline
+  - your self-check with a preliminary evaluation that is based on the evaluation criteria from the task
+- use _gh-pages_ for deployment
+- submit **a link to deploy** in Cross-Check: Submit tab
+
+## Technical requirements
+
+- the application is checked in the latest version of Google Chrome browser
+- jQuery is not allowed
+- you can not use Angular / React / Vue / Typescript
+- you can use bootstrap, CSS frameworks, HTML and CSS preprocessors
+
+## Criteria for evaluation:
+
+**Maximum score for the task: 250 points**
+
+### Basic scope +80 points
+
+- [ ] layout, design, responsive UI: `+20`
+- [ ] at the beginning state of the game, the frame has size 5x5. The sequence of numbers is logically arranged and help the player solve the nonogram: `+20`
+- [ ] every 5 cells in a rows and columns (both player's field and clues) should be divided by divider `+5`
+- [ ] when user clicks on cells using mouse left-click - it should be mark as dark. When user click on dark cell - it should be mark as empty (white) cell: `+15`
+- [ ] the game should end when the player reveals all cells correctly and related message is displayed at the end of the game: `+20`
+
+### Advanced scope +90 points
+
+- [ ] the game should have at least 5 templates for easy level (5x5) and the player is able to choose what picture he/she wants to solve. `+15`
+- [ ] a player is able to fill in a square in the grid changing the color of the grid to crossed-cell(X) using right mouse-click. Context menu doesn't appear: `+20`
+- [ ] the game can be restarted without reloading the page: `+15`
+- [ ] game duration is displayed, stop-watch will start after first click on field (not on clues): `+10`
+- [ ] sound accompaniment (on/off) for every events (see **Advanced** block): `+15`
+- [ ] implemented saving the state of the latest game: `+15`
+
+### Hacker scope +80 points
+
+- [ ] option to choose different themes for the game board (dark/light themes): `+15`
+- [ ] ability to change the size (5x5, 10x10, 15x15) is implemented and there are least 5 templates for each level: `+20`
+- [ ] implemented saving the latest 5 win results with sorting: `+15`
+- [ ] "random game" button is implemented: `+15`
```

Comment:
I think here we need to write not only a button but also the logic of a random game should be implemented.